### PR TITLE
91752: Add field Title to Customer object

### PR DESCRIPTION
**Pull request by jpetrzilka**

### CODE REVIEW

File: mews-api.md [hunk 1]
```diff
@@ -917,6 +919,7 @@ There are certain rules that need to be followed in order for Mews to process th
 | `email` | `string` | _recommended_ | Email. |
 | `lastName` | `string` | required | Last name. |
 | `firstName` | `string` | optional | First name. |
+| `title` | `string` | optional | Accepted values: `mister`, `misses`, `miss` |
```
Comment (patrikmiksovsky95) [inline mews-api.md:922]: Values are uppercase. + We extract this as separate enum check https://mews-systems.gitbook.io/connector-api/operations/reservations#reservation-state in connector API they mark it as string with types and linked to that type
e.g. https://mews-systems.gitbook.io/connector-api/operations/reservations#reservation-parameters reservation state
+ you are missing changelog

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1 March 2022 12:00 UTC
+
+* [Customer](mews-api.md#Customer) object extended. The field `title` was added.
```
Comment (MichalVacek) [inline changelog.md:5]: From the structure below it seems that the message should be a little different. Based on line 90 (## 26th March 2020 12:00 UTC)
```suggestion
* [Customer](mews-api.md#Customer) extended. The `title` is now supported.
```

**Review by MichalVacek — APPROVED** [review]

File: mews-api.md [hunk 3]
```diff
@@ -760,6 +760,7 @@ There are certain rules that need to be followed in order for Mews to process th
         "email": "john@smith.com",
         "firstName": "John",
         "lastName": "Smith",
+        "title": "mister",
```
Comment (patrikmiksovsky95) [inline mews-api.md:763]:
```suggestion
        "title": "Mister",
```

File: mews-api.md [hunk 4]
```diff
@@ -826,6 +827,7 @@ There are certain rules that need to be followed in order for Mews to process th
                     "email": "jane@smith.com",
                     "firstName": "Jane",
                     "lastName": "Smith",
+                    "title": "misses",
```
Comment (patrikmiksovsky95) [inline mews-api.md:830]:
```suggestion
                    "title": "Misses",
```

File: mews-api.md [hunk 5]
```diff
@@ -917,12 +919,19 @@ There are certain rules that need to be followed in order for Mews to process th
 | `email` | `string` | _recommended_ | Email. |
 | `lastName` | `string` | required | Last name. |
 | `firstName` | `string` | optional | First name. |
+| `title` | `string` [Title](mews-api.md#titles) | optional | Title. |
```
Comment (patrikmiksovsky95) [inline mews-api.md:922]:
```suggestion
| `title` | `string` [Title](mews-api.md#customer-title) | optional | Title. |
```

File: mews-api.md [hunk 6]
```diff
@@ -917,12 +919,19 @@ There are certain rules that need to be followed in order for Mews to process th
...
 | `telephone` | `string` | optional | Telephone. |
 | `loyaltyCode` | `string` | optional | Loyalty code of the customer. |
 | `nationalityCode` | `string` | optional | [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) - two letter country code or [ISO 3166-1 alpha-3 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) - three letter country code. |
 | `languageCode` | `string` | optional | Language [code](https://msdn.microsoft.com/en-us/library/ee825488) of the communication language of the customer. This language will be used as the default for communication with the customer. |
 | `address` | [`Address`](mews-api.md#address) object | optional | Represents address. |
 
+#### Titles
```
Comment (patrikmiksovsky95) [inline mews-api.md:929]:
```suggestion
#### Customer title
```

**Pull request merged**